std/math.d

```diff
@@ -8314,27 +8325,89 @@ if (isFloatingPoint!(F) && isFloatingPoint!(G))
     return impl(x, y);
 }
 
+/*
+Of course, floating point numbers should be compared using isClose,
+but the main purpose of this test is to provide a good example,
+not to test the pow function, and this would lead to ugly examples like:
+
+writeln(isClose(pow(1.5, 10.0), 57.6650390625)); // true
+
+instead of
+
+writeln(pow(1.5, 10.0)); // 57.6650390625
+
+So I versioned this unittest out, because on some 32bit computers the tests
+would fail with ==.
```

Comment:
That's exactly what the dscanner tests tries to prevent people from doing. This will lead to user confusion if they copy this code to user machines and it fails there. Either:
- use is close
- use `version (X86_64)`  inside the test, so that it's obvious to the user what's going on.